WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -6222,16 +6221,22 @@
 	}
 }
 36466 = {
-	name = "Abd Allah"
-	# Abd Allah ibn 'Uthman banu Taym
+	name = "Abu Bakr"
+	# Abu Bakr ibn 'Uthman banu Taym
 	dynasty = 1042501
 	religion = "semitic_pagan"
 	culture = "adnanite"
 	father = 36465
 	573.1.1 = {
 		birth = yes
 	}
-	634.1.1 = {
+	610.1.1 = {
+	    religion = islamic
+	}
+	632.6.8 = {
+	    religion = sunni
+	}
+	643.8.23 = {
```

Comment:
Just noticed what seems to be a vanilla error. Should it's be 634 instead of 643? The rest is OK.

Reply:
Yes, i really not thought that vanilla version had such error.

Reply:
Probably just a typo on their side, the month and day are correct.

```suggestion
	634.8.23 = {
```